apps/core/factories.py

```diff
@@ -1,10 +1,18 @@
 from factory import Faker, Iterator, SubFactory
 from factory.django import DjangoModelFactory
+from django.utils import timezone
 
 from apps.data.factories import EntryFactory, RepositoryFactory
+from faker import Faker as faker_Faker
 from . import models
 
 
+def Get_date_with_timezone():
```

Comment:
Naming convention for functions is lower case with words separated by underscores (see [PEP8 reference](https://www.python.org/dev/peps/pep-0008/#function-names)).

Reply:
Ok. Thank you. Why didn't flake8 say anything? Strange...